server/common/pom.xml

```diff
@@ -250,6 +250,18 @@
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>${maven.jacoco.plugin.version}</version>
+                <configuration>
+                    <excludes>
+                        <exclude>**/org/eclipse/lsp/cobol/common/model/**/*</exclude>
```

Comment:
Why do we ignore them?

Reply:
These are the model classes which doesn't have any logic ( at least I assumed so  )

Reply:
I have doubt about this implementation as well. Was thinking should we go for a more granular coverage exclude logic. Or do you think we shouldn't excluded sources from test coverage ? If so, I completely understand. Will close this PR in that case

Reply:
make sense. I will close this PR 👍